NetProbe/Model/NetworkMeasurement.mm

```diff
@@ -53,21 +54,69 @@ @implementation NetworkMeasurement
 
 -(id) init {
     self = [super init];
-    self.logLines = [[NSMutableArray alloc] init];
-    self.finished = FALSE;
+    NSBundle *bundle = [NSBundle mainBundle];
+    geoip_asn = [bundle pathForResource:@"GeoIPASNum" ofType:@"dat"];
+    geoip_country = [bundle pathForResource:@"GeoIP" ofType:@"dat"];
+    ca_cert = [bundle pathForResource:@"cacert" ofType:@"pem"];
+    self.completed = FALSE;
     return self;
 }
 
 -(void) run {
     // Nothing to do here
 }
 
--(NSString*) getDate{
+-(NSString*) getDate {
     NSDateFormatter *dateformatter=[[NSDateFormatter alloc]init];
     [dateformatter setDateFormat:@"dd-MM-yyyy HH:mm:ss"];
     return [dateformatter stringFromDate:[NSDate date]];
 }
 
+-(NSString*) getFileName:(NSString*)ext {
+    NSArray *paths = NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES);
+    NSString *documentsDirectory = [paths objectAtIndex:0];
+    //double timestamp = mk::time_now();
+    NSString *fileName = [NSString stringWithFormat:@"%@/test-%@.%@", documentsDirectory, self.test_id, ext];
+    //NSLog(@"fileName %@", fileName);
+    return fileName;
+}
+
+-(void)writeOrAppend:(NSString*)string{
+    NSString *fileName = [self getFileName:@"txt"];
+    NSFileManager *fileManager = [NSFileManager defaultManager];
+    if(![fileManager fileExistsAtPath:fileName])
+    {
+        [string writeToFile:fileName atomically:NO encoding:NSStringEncodingConversionAllowLossy error:nil];
+    }
+    else
+    {
+        NSFileHandle *myHandle = [NSFileHandle fileHandleForWritingAtPath:fileName];
+        [myHandle seekToEndOfFile];
+        [myHandle writeData:[[NSString stringWithFormat:@"\n%@",string] dataUsingEncoding:NSUTF8StringEncoding]];
+    }
```

Comment:
I discussed this with @lorenzoPrimi and we agreed that for now we will leave this code in place, since it works, but that it would be optimal for MK to save logs (see measurement-kit/measurement-kit#706).